### PR TITLE
How AIOps is transforming IT operations

### LINKED ISSUE #865: How AIOps is transforming IT operations
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

Digital transformation encompasses DevOps and the adoption of cloud and new technologies like containers. It represents a shift from centralized IT to applications and developers, an increased pace of innovation and deployment, and the acquisition of new digital users—machine agents, Internet of Things (IoT) devices, Application Program Interfaces (APIs), etc. All of these new technologies and users are straining traditional performance and service management strategies and tools to the breaking point. AIOps is the IT operations solution required to handle these digital transformation issues. 
### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?

1. What is AIOps
2. Benefits and challenges of AIOps 
3. How AIOps works 
4. Best AIOps software 

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)
N/A

### PR DESCRIPTION
**Pull request by EKahuha**
this fixes #865

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

File: articles/how-aiops-is-transforming-it-operations/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+It does this by absorbing data in the form of metrics, events, and logs and taking the data through a set of algorithms to select specific data points. These data points help to identify data patterns and correlations and draw inferences. The inferences are then shared across departments, thus promoting a collaborative work environment.
+
+#### Fast data processing
+
+It takes a lot of time for humans to process large amounts of data. AIOps uses algorithms powered by big data and machine learning to derive cognitive insights from raw datasets.
+
+AIOps reduces metrics such as the [Mean Time to Repair (MTTR) and the Mean Time to Detect (MTTD)](https://resolve.io/writable/documents/AIOps_For_Dummies-Nov19.pdf) dramatically. MTTR represents the amount the IT team takes to eradicate, remediate, or control the organization&#39;s already discovered threats. MTTD is the amount of time the IT team takes to notice a potential threat. The ability to process data at light fast speed saves IT operations (ITOps) teams&#39; time, effort and reduces the risk of operational fatigue.
```
Comment (lalith1403) [inline articles/how-aiops-is-transforming-it-operations/index.md:39]: amount of time? Specify that.

File: articles/how-aiops-is-transforming-it-operations/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+#### Data-driven decision making
+
+By an organization adopting AIOps, it introduces important ML techniques to its IT operations. These include predictive analysis, pattern matching, casual analysis, and historical data analysis. With these techniques, an organization makes decisions primarily driven by data and provides automated responses to incidences. This eliminates data noise and human error. Data with a lot of meaningless information is referred to as [noisy data](https://en.wikipedia.org/wiki/Noisy_data#). Such kind of data is challenging to understand and interpret.
+
+#### Eliminates data silos
+
+Data silos is a common problem for many organizations and a source of inefficiency. Organizations faced with [data silos](https://www.plixer.com/blog/data-silo-what-is-it-why-is-it-bad/#) find that only particular employees or one group of employees has access to the source of data or a specific set of data in the organization. In such a situation, you will find multiple teams storing the same data or complementary data separately. This leads to wasted resources in terms of storage costs and inhibited productivity.
```
Comment (lalith1403) [inline articles/how-aiops-is-transforming-it-operations/index.md:29]: What is data silos? Can you elaborate on that?

**Review by lalith1403 — CHANGES_REQUESTED** [review]

**Comment by EKahuha** [comment]
@lalith1403 
file updated

**Review by lalith1403 — APPROVED** [review]

**Comment by hectorkambow** [comment]
👋  Good afternoon @EKahuha  

Giving your article a final review - its looking really good so far upon a quick glance. 

Wanted to ask if we could add a short sentence for the list you provide at the end - I think it would really add value to the reader and community to get a brief description of the AIops software you reference. 

Great job adding the links as well. Really helpful 🚀

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by EKahuha** [comment]
Good morning @hectorkambow 
I have updated this article. Please have a look.

**Pull request merged**